src/main/java/org/bukkit/World.java

```
@@ -173,6 +174,28 @@
     public void loadChunk(int x, int z);
 
     /**
+     * Loads the {@link Chunk} at the specified coordinates then runs the
```

This entire JavaDoc looks like it needs to be reworded. Punctuation and more complete sentences should be used as this currently is more or less in bullet point form with periods.

There should be zero confusion to a plugin author as to what this does, what can and cannot happen, and what happens if something goes wrong (eg: a null callback). The author should also be able to see what is REQUIRED and what is not required (such as in the case of the callback - What happens if I give it null? Is it required?).

Is it also the best idea to notify the callback if the chunk is already loaded? I personally would rather see some sort of error or a clear indication that I, as an author, am trying to load something that already exists. (Or should we silently fail like in the existing load chunk method?)